charts/ia-case-documents-api/values.yaml

```diff
@@ -13,18 +21,9 @@ java:
         - system-password
         - idam-client-id
         - idam-secret
-        - idam-redirect-uri
```

Comment:
there is also `bootstrap.yaml` file to clear

Reply:
removed